### PR TITLE
Upgrade to Traefik 2

### PR DESCRIPTION
**Pull request by alexlanz**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -10,135 +10,83 @@ This project uses Traefik as a reverse proxy and provides already a predefined s
...
-
-### Stop Traefik
-
-```bash
-docker-compose down
-```
-
 ## Documentation
 
 Additional information about the usage of Traefik with Docker can be found [here](https://docs.traefik.io/configuration/backends/docker/). There you can find additional configuration settings and possible security concerns, that should be considered.
```
Comment (stplasim) [inline readme.md]: This link is broken. The latest link to the documentation for docker is this one (https://doc.traefik.io/traefik/providers/docker/)

Comment (alexlanz) [reply readme.md]: Thx, I removed the documentation section now entirely.

**Review by stplasim — APPROVED** [review]
Looks good

**Comment by mmalfertheiner** [comment]
LGMT 💯

**Review by mmalfertheiner — APPROVED** [review]

**Pull request merged**